doc/release/1.12.0-notes.rst

```diff
@@ -261,6 +261,11 @@ values. There are two different propagation modes. The default causes masked
 values to contaminate the result with masks, but the other mode only outputs
 masks if there is no alternative.
 
+Array dimension functions generalized with new ``atleast_nd``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This should be moved to 1.13.0-notes and changed to `-------` next time it is rebased